model/src/test/kotlin/PackageCurationTest.kt

```diff
@@ -387,6 +387,34 @@ class PackageCurationTest : WordSpec({
                 result.metadata.declaredLicensesProcessed.spdxExpression shouldBe expression
             }
         }
+
+        "preserve OR operator when mapping makes previously invalid licenses valid" {
```

Comment:
Just as a remark to begin with / as a note to myself: We cannot generally assume that `OR` needs to be preserved in this test, as the operator is package-type-specific.